data-table/table.js

```diff
@@ -149,12 +149,68 @@ function onColumnEntryBoxEnter(event, boxValue) {
      */
 }
 
-function addColumns(numberOfColumns, index) {
-    /**
-     * Adds columns from existing table. Will probably call
-     * createEntryCell(). Maybe should also accept index as an argument?
-     * TODO: Fill this out
-     */
+
+/**
+ * Adds colunms to the table.
+ * @param {object} config       - Table configuration object
+ * @param {Number} numberOfColumns - Number of columns to be added
+ * @returns {undefined}         - Doesn't return anything
+ */
+function addMultipleColumns(config, numberOfColumns) {
+    for(let i = 0; i < numberOfColumns; i++){
+        addSingleColumn(config, i);
+    }
+}
+
+/**
+ * Adds a single colunm to the table
+ * @param {object} config   - Table configuration object
+ * @returns {undefined}     - Doesn't return anything
+ */
+function addSingleColumn(config){ 
+    let table = document.getElementById(config.tableIds.tableElementId);
+    let numRows = table.rows.length; // get length row right now
+    let numCols = table.rows[0].cells.length;
+    for(let i = 0; i < numRows; i++){
+        if(i == 0){  // head 
```

Comment:
Like we talked about on Zoom, everything in this for loop can be replace by `createEntryCell(config, table.rows[i], i, config.numColumns, " (" + i + ", " + config.numColumns + ") ")`